src/main/java/org/janelia/saalfeldlab/paintera/control/actions/AllowedActionsProperty.java

```diff
@@ -0,0 +1,81 @@
+package org.janelia.saalfeldlab.paintera.control.actions;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.value.ChangeListener;
+import javafx.beans.value.ObservableValue;
+import javafx.scene.Cursor;
+import javafx.scene.Node;
+
+public class AllowedActionsProperty extends SimpleObjectProperty<AllowedActions> {
```

Comment:
Extracted to address https://github.com/saalfeldlab/paintera/issues/427

allows for `disable()` and `enable()` which will disable UI elements, and set the cursor to `WAIT`